cloud/pkg/client/clientset/versioned/fake/register.go

```diff
@@ -18,13 +18,14 @@ limitations under the License.
 package fake
 
 import (
-	devicesv1alpha2 "github.com/kubeedge/kubeedge/cloud/pkg/apis/devices/v1alpha2"
-	reliablesyncsv1alpha1 "github.com/kubeedge/kubeedge/cloud/pkg/apis/reliablesyncs/v1alpha1"
```

Comment:
These files are genetared by client-gen, and can't be modified manually, haven't the ci returned error?

Reply:
Double checked, verify code-gen now doesn't work, maybe related to the dependency upgrade

Reply:
Those files would be ignored.